c-pallets/file-bank/src/lib.rs

```diff
@@ -499,7 +509,7 @@ pub mod pallet {
 			filler_list: Vec<FillerInfo<T>>,
 		) -> DispatchResult {
 			let sender = ensure_signed(origin)?;
-			if filler_list.len() > 10 {
+			if filler_list.len() > 10 { // Pass this limit to a const or configurable const in the pallet
```

Comment:
Now I set a constant for it:
https://github.com/CESSProject/cess/pull/88/files#diff-9c37b1a61a6e9394aac1e9383bf9d3fac9539378d1af8fcd75aed4f8a4f42967R521-R522